go.mod

```diff
@@ -573,6 +573,11 @@ require (
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
 
+require (
```

Comment:
Nit: this should be moved to the top grouping and have a comment with the "owners" of the dep